### DISCUSSION TITLE
Please add The Function to Rename Dictionary

### DISCUSSION THREAD

**Discussion by shcfan** [xiaoyifang/goldendict-ng#1079] [Ideas]
To same as the snapshot. Thanks!
![2023-08-24_105745](https://github.com/xiaoyifang/goldendict-ng/assets/78716981/97c928b9-91da-4a56-a641-51d262b508e2)

**Comment by xiaoyifang** [comment]
https://xiaoyifang.github.io/goldendict-ng/custom_dictionary/#:~:text=zip%0A%20%20%20%20%20%E2%94%94%E2%94%80%E2%94%80%20Collins.dsl-,Override%20display%20name,-Some%20dictionary%20formats


already supported

**Reply by shcfan** [reply]
Thanks for your replying.
